src/pydata_sphinx_theme/assets/styles/sections/_sidebar-secondary.scss

```diff
@@ -27,15 +27,14 @@
 .bd-toc {
   -ms-flex-order: 2;
   order: 2;
-
-  height: calc(100vh - 2rem);
+  max-height: calc(100vh - 2rem);
   overflow-y: auto;
 
   @supports (position: -webkit-sticky) or (position: sticky) {
```

Comment:
I kinda feel like we could remove the `@supports` conditional here, since all major modern browser support `position: sticky`: https://caniuse.com/css-sticky

Reply:
agreed (I also did it for the primary navbar)